### ISSUE TITLE
COSMIC Desktop

### ISSUE THREAD

**Issue by EdenQwQ** [catppuccin/catppuccin#2373]
### Link to repository

https://github.com/EdenQwQ/cosmic

### Screenshots

![](https://github.com/EdenQwQ/cosmic/raw/main/assets/preview.webp)

### Any additional comments?

COSMIC Desktop is a new desktop environment developed by the PopOS team. The new DE is about to have its first alpha release. This repository offers catppuccin themes for both the desktop and cosmic-terminal.

### Submission Guidelines

- [X] I have read and followed Catppuccin's submission guidelines.

**Comment by EdenQwQ** [comment]
> Just took a quick look at it. Looks like you need to update the links to the catppuccin ones and in the title of the README point to some page that explains the COSMIC DE or something along those lines :)

Updated. Now the link points to https://github.com/pop-os/cosmic-epoch.

**Comment by justTOBBI** [comment]
Don't forget to update the install instructions to match the future catppuccin repository :)

**Comment by EdenQwQ** [comment]
> Don't forget to update the install instructions to match the future catppuccin repository :)

Updated. Should I also update the shields.io links for now?

**Comment by justTOBBI** [comment]
> > Don't forget to update the install instructions to match the future catppuccin repository :)
> 
> Updated. Should I also update the shields.io links for now?

Pointing them to the catppuccin repo should do

**Comment by EdenQwQ** [comment]
> Pointing them to the catppuccin repo should do

Updated.

**Comment by sgoudham** [comment]
Hey :wave:, sorry for the long wait on the review for this @EdenQwQ 

The repository and theme looks great! Thanks for taking the time to do it.

Just a few comments about the repository:

1. Please fix the [spelling mistake of "Catppuccino"](https://github.com/EdenQwQ/cosmic/blob/1b88e73d2f753aedceaa1f491b574d3e79520463/generate.py#L5) to Catppuccin. Note the help usage in the README will also need to be updated.
	
2. We should pin the version of the `catppuccin` python package in the [requirements.txt](https://github.com/EdenQwQ/cosmic/blob/main/requirements.txt#L1) to avoid unexpected breakages when new versions are published. 

Happy to approve the repository after the above comments are resolved :+1:

**Comment by EdenQwQ** [comment]
@sgoudham Thanks for replying. All of the comments should be resolved in the latest commit.

**Comment by sgoudham** [comment]
Thanks @EdenQwQ! 

Since that comment, I received some opinions on discord for the name and while it's not necessarily a big deal. **@backwardspy and I eventually concluded that we should transfer it under `catppuccin/cosmic-desktop`**. The rule is usually the "shortest name" possible but we didn't feel like `cosmic` or `cosmic-de` truly captures what this port is for. (Our naming conventions are a bit all over the place anyways :smile:)

Don't want to block this any further for some README updates so I'm happy to approve and get this merged into the organisation!

Please read, ["Transferring a repository owned by your personal account"](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository#transferring-a-repository-owned-by-your-personal-account), if you are unsure about the process of transferring the repository to Catppuccin.

**We'll close this issue once the port has been added to our [ports.yml](https://github.com/catppuccin/catppuccin/blob/main/resources/ports.yml)**.
